app/Models/Contest.php

```diff
@@ -54,4 +54,14 @@ public function getCSVExport()
 
         return buildCSV($data);
     }
+
+    /**
+     * Set the sender attribute for the contest.
+     *
+     * @param string $value
+     */
+    public function setSenderAttribute($value)
+    {
+        $this->attributes['sender'] = ! empty($value) ? $value : null;
```

Comment:
oh my bad. was confused cause i didn't see a migration for that field. was that in a different PR?